### ISSUE TITLE
Removal of money orders

### ISSUE THREAD

**Issue by keo-bisq** [bisq-network/proposals#75]
> _This is a Bisq Network proposal. Please familiarize yourself with the [submission and review process](https://docs.bisq.network/proposals.html)._

I suggest that money orders (MO) should be removed as payment option. They are impossible to arbitrate in case they are stolen and cashed by somebody else than the intended recipient. There are virtually no checks on the person cashing them. In addition the sender of the MO can only provide easily forged photos of the MO and the envelope. They are in addition seldom used.

**Comment by m52go** [comment]
How widespread of an issue is this? Have arbitrators faced these issues frequently enough for it to be a problem? Are we talking just about US Postal Money Orders, or other money orders too?

I'm fine with considering this proposal positively if there is enough of a reason, but workable fiat payment methods are hard to find so we should be make sure the reasons for removing one are solid.

**Comment by ManfredKarrer** [comment]
To me it sounds like a very unsafe method and as it was not used much anyway I would prefer to remove it.

**Comment by justpaulgmx** [comment]
Agreed, let's pull this payment method now before someone reads this thread and exploit the weakness. It's worried me for some time. Any objections to closing this?

**Comment by ManfredKarrer** [comment]
Can all give a thumbs up or down?

**Comment by HarryMacfinned** [comment]
year | US Postal MO
-----|-------------
2019 | 41
2018 | 42
2017 | 12
2016 | 3

So, MO was indeed seldom used in the past years,
but for now, 2019 Q1 has already more MO than 2018.

And I would also like to know if we talk just about US Postal Money Orders, or other money orders too?

**Comment by justpaulgmx** [comment]
Thumbs up to removing money orders. When thinking of arbitration, we must think of scailability as use increases exponentially. At minimum it is a tough risky arbitration transaction, at worst an attack vector on the DAO. Just my 0.02 sats.

**Comment by clearwater-trust** [comment]
I am not in favor of removing MO's. One difficult dispute is not enough data to remove the payment method altogether.

**Comment by huey735** [comment]
I was completely ignorant to Money Order up until 3 days ago and I have no idea how widespread its use is. But I'm sympathetic to the argument that we should do our best to maintain the Cash payment methods.

@HarryMacfinned where did you get those numbers?

**Comment by alexej996** [comment]
I think it might be better that it is grouped with F2F offers in some way as trades that can not be arbitrated.

**Comment by m52go** [comment]
> I am not in favor of removing MO's. One difficult dispute is not enough data to remove the payment method altogether.

> I think it might be better that it is grouped with F2F offers in some way as trades that can not be arbitrated.

I agree with these statements. Maybe we just need to make the risks of using money orders more clear, i.e., that people are on their own with money orders, just like they are with F2F.

Additionally: many of us are concerned about privacy, but what about the unbanked? A money order is the only practical way for Bisq to work for such people.

There are a million things that could go wrong with F2F, but that that didn't stop us from adding it. I don't like the idea of preemptively removing functionality out of hypothetical fear.

**Comment by clearwater-trust** [comment]
This may be related, so I'm going to add it here. I don't think arbitrators should be personally liable for each of their disputes. I've heard it mentioned that arbitrators are currently taking on the risk of each dispute. This should be the DAO's responsibility, not the arbitrators. Else we end up with very few payment options :smile: 

If the dispute is decided in error, the DAO should vote on reimbursement. If the arbitrator is malicious their bond should be burned. If the arbitrator's performance is poor, their privileges will be removed. I think this will provide a more agnostic perspective from the arbitrator.

**Comment by ManfredKarrer** [comment]
I think F2F and MO has different circumstances. 

Scamming someone in a F2F trade carries lots of risk for the scammer as well. 
Scamming someone with MO seems riskless for the scammer.

If we know that there is a method how to scam the peer without taking risk I think it is not a good idea to just delegate it to the user and assume they fully understand the risk. 

In reality users often start a trade and don't understand really how even Bisq works. To fully understand the risk situation with a payment methods requires much more experience. I certainly did not had a clue  about such risks before I started working on Bisq and when I did my first LBTC trade I also did not fully understand the risks and the system. 
A user who got scammed will likely be a frustrated Bisq user, never coming back and not recommending Bisq in the best case and in the worst case he will make negative prototion that there is lots of scam going on at Bisq (as it is the case with LBTC). LBTC also has warnings and educational material about risks, etc. but the fact that there is a high scam rate falls back to them and many people have the impression LBTC sucks as there are too many scammers, even those might mostly happen with payment methods which are marked as high risk like Paypal. Also having one payment methods where scam is systematically possible will attract scammer and then they will explore other options as well. Prefer to not give them any honey to attract them in the first place.

I am also not so sure how much privay protection MO really delivers as I heard that it requires ID verification. But I don't know details...
We would still need a payment method expert who really understands all the context and can give solid advice.

We might have new options once the DAO is out, like using a bond for reputation. That would work pretty good with MO as there is no chargeback risk but only a risk in the short timer period of the trade. So only for that period the bond need to be able to cover the trade amount. Still would require more as a scammer could make several trades in parallel and the total trade sum might not be covered by the bond anymore. 

Another option is burned BSQ (proof of burn) to create reputation one will not want to lose by getting marked as a scammer.... 

Too early to think on concrete implementations of those options but I think those will enable higher risk payment methods without losing security.

**Comment by justpaulgmx** [comment]
Changing status to same as F2F seems to solve the original issue of arbitration liability  and is a solid argument. Concerns for access by the unbanked should also be weighed. I'm positive about the suggestions.

**Comment by ManfredKarrer** [comment]
As said I would not compare it with F2F as security in F2F has other context (real life physical access) then the anonymous nature of MO. F2F was also added with the condition to see how it works and if there would be too many issues we would remove it.

**Comment by HarryMacfinned** [comment]
@huey735 wrote:
> where did you get those numbers?

I get in the Market/Trades tab.
There you can order the payment-method column.
And I counted manually.

It's perhaps also possible to extract such numbers with the webapi. But I'm not 100% sure.

**Comment by rainysam** [comment]
I would strongly recommend keeping money orders as an option of payment. It has been, and is, the only payment method I've used when making fiat BTC purchases through Bisq. I would prefer face-to-face (F2F), but I don't live in a populated enough area that other Bisq users are near me. I've made a dozen or so purchases throughout 2018 via MOs without any issues. However, my most recent purchase (January) is still under arbitration, and I believe is the impetus for this proposal.

Among all of the options to purchase with fiat, MOs are the most private. They don't require any ID to purchase and are purchased with cash (or maybe debit cards, although I haven't used them for this purpose). I've never cashed a MO myself, and I am unaware of KYC requirements on that end. I'd imagine that there isn't much scrutiny when someone's cashing a postal MO, as they're valid for $1000 US max and there's no default risk on the person who's cashing it (unlike a personal check, which will bounce if there's not enough money in the account). They're kind of like a cashier's check, backed by the issuer (USPS, Western Union, et al) rather than someone's personal or business account. MO's are probably the financial instrument most closely related to physical cash, as they're well recognized, difficult to counterfeit, and act as bearer bond essentially. When sent in the mail, it's sort of like sending cash, but with the added safeguard of being able to be replaced if it's lost (and not cashed by someone else). The USPS won't do anything for you if it is cashed by someone else, and will suggest that you pursue remedies through the legal system. 

I would definitely be in favor of more clearly defining the risks, so that potential future users are aware. I was aware of the risks when I started making MO purchases, and figured that it was a matter of time before someone tried to pull a fast one on me. That's the risk of using such a system. If I didn't care about supporting lousy companies or my privacy, I could make my purchases on Coinbase and save money on fees (fees being cost of shipping MO's and potential fraud risk) and hassle (going to post office and potential fraud), but I feel the extra cost in terms of fees and hassle factor are preferable to using other services.

 I'd say it's still less risky than F2F, when you consider the risk potential for violence.

Thanks for all of your great work, devs. Keep it up!

**Comment by keo-bisq** [comment]
For me as an arbitrator it is enough that a popup or something is introduced clearly saying that people receiving MOs must be aware that they can be stolen, that Bisq cannot bear any responsibility for this and that the BTC will be awarded to the sender. In addition I think the name of the receiver of the MO should be written on it to make it more difficult to cash for someone else. **rainysams** arguments are quite strong and now I think MOs should be kept as a payment option. If further problems pop up with MOs they can be deleted later. The frequency of disputes involving MOs is around 1% of all MO-trades and their use is increasing.

**Comment by ManfredKarrer** [comment]
@rainysams , @keo-bisq 
Could you provide a suggestion for a popup text? You can post it here...

**Comment by keo-bisq** [comment]
A suggestion:
"Money orders are less safe, but more private than e. g. bank transfers. Bisq will not bear any responsibility in case a sent money order is stolen, and the arbitrators will in such cases award the BTC to the sender of the money order (who has to show that the money order was sent and received). In order to minimise the risk that the money order is cashed by somebody else than the buyer the sender should write the buyers name on the money order."

This text should be reviewed by somebody who has used money orders. In particular I am not sure if the sellers name should be written on the MO since the seller might want to sell the MO for increased privacy. I am OK with and without the last sentence.

**Comment by HarryMacfinned** [comment]
@rainysam , 
when you use a MO to send fiat money to somebody,
I presume that you write the receiver's postal address on the envelope ?
Or can you use also poste restante service (https://en.wikipedia.org/wiki/Poste_restante)
(sorry, I never used MO so far).

**Comment by rainysam** [comment]
@HarryMacfinned When you set up a method of payment on Bisq, you have to specify a name & address. That info is presented to the BTC purchaser when he initiates the trade. I suppose you could use poste restante, you'd just have to use that address information when you set up the MO profile on Bisq. Bisq sets an 8-day window of payment; that is, once the MO trade is initiated, the trade goes on to arbitration automatically if the seller doesn't confirm receipt of the payment within 8 days.

**Comment by rainysam** [comment]
@ManfredKarrer I do like the suggestion of @keo-bisq 

Regarding writing the seller's name/address on the MO: I've not written that info in because it allows for more privacy for the seller. When setting up the MO fiat account in Bisq, as a buyer, there's no reason that one can't use a pen name if preferred. As a seller, however, if you were to use a pen name, and that name was written on the MO, it'd potentially make it more difficult to cash. I've not ever cashed a money order, but I know they can be cashed at the USPS, check cashing stores, and banks. I'd imagine that banks would only cash one for you if you have an account with them, and they'd be hesitant to cash a MO with someone else's name on it. I'm not sure if USPS or check cashing stores require ID at all.

Here's my recommendation of text for a pop-up message (I copied and pasted some parts from @keo-bisq):

"Money orders are one of the more private fiat purchase methods available on Bisq. However, please be aware of potentially increased risks associated with their use. Bisq will not bear any responsibility in case a sent money order is stolen, and the arbitrators will in such cases award the BTC to the sender of the money order, provided they can produce tracking information and receipts. It may be advisable for the sender to write the BTC seller's name on the money order, in order to minimize the risk that the money order is cashed by someone else."

**Comment by ManfredKarrer** [comment]
Added the warning text.
